frontend/src/components/CriteriaIcon.tsx

```diff
@@ -15,9 +15,6 @@ const CriteriaIcon = ({
   return (
     <Box
       sx={{
-        '& img': {
-          display: 'block',
-        },
```

Comment:
This is not critical, but removing this style breaks the alignment when the icon is used besides a text:
![Peek 03-05-2022 13-28](https://user-images.githubusercontent.com/4726554/166445211-7fb6a9c1-57cf-4260-8308-82da978a01ab.gif)


It would be cleaner to keep this as "block", and apply "flex" styling on the parent component where this icon is used:
```
display: flex,
align-items: center
```

Reply:
Fixed by https://github.com/tournesol-app/tournesol/pull/860/commits/34c3cd08abb100f2195dd0f6e1e930f7cb6f770a.